fs/walk.ts

```diff
@@ -520,8 +520,8 @@ export async function* walk(
  * Recursively walks through a directory and yields information about each file
  * and directory encountered.
  *
- * The file paths are absolute paths. The root directory is included in the
- * yielded entries.
+ * The root path determines whether the file paths is relative or absolute.
```

Comment:
```suggestion
 * The root path determines whether the file paths are relative or absolute.
```

Reply:
This mistake was fixed before merging :D 
-> https://github.com/denoland/std/pull/5559/commits/a434f65f551cdfde4d4b74c075e11d785c0d93e7

Reply:
@ryuapp You're absolutely right — I'm not sure why I wasn't shown the changes from that commit when I clicked to review… looks good!